squad/core/utils.py

```diff
@@ -130,3 +131,37 @@ def split_list(_list, chunk_size=1):
         chunks.append(_list[:chunk_size])
         _list = _list[chunk_size:]
     return chunks
+
+
+def _log_entry(request, object, message, flag):
+    from django.contrib.auth.models import AnonymousUser
+    from django.contrib.contenttypes.models import ContentType
+    from squad.http import auth_user_from_request
```

Comment:
Is there an import loop? Why not put these on the top?

Reply:
yes, there is. I didn't figure out how to avoid it in some other way.